docs/docs/app/dimensions.mdx

```diff
@@ -7,15 +7,15 @@ slug: dimensions
 
 # Dimensions
 
-Dimensions let you drill down into experiment results. For example, if you define a `browser` dimension, you can
+Dimensions let you drill down into experimental results. For example, if you define a `browser` dimension, you can
```

Comment:
In this case we are referring to the experiment resource so changing this to the adjective wouldn't make sense.